gulp/tasks/test.js

```diff
@@ -2,11 +2,13 @@
 
 var gulp = require( 'gulp' );
 var $ = require( 'gulp-load-plugins' )();
-var protractor = require( 'gulp-angular-protractor' );
-var exec = require( 'child_process' ).exec;
+var childProcess = require( 'child_process' );
+var exec = childProcess.exec;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/b8bd3951ea64008d12866b4ba53c442f278ac687